src/components/reusable/dropdown/index.tsx

```diff
@@ -85,7 +91,13 @@ const Dropdown = ({
   };
 
   return (
-    <div className="relative w-full">
+    <div
+      className={`relative w-full ${wrapClassName}`}
+      onClick={(e) => {
```

Comment:
`e` doesn't sounds good, name it `event` for better readability

Reply:
Fixed it.